budget_manager/models.py

```diff
@@ -1,3 +1,41 @@
 from django.db import models
 
 # Create your models here.
+
+class Sources(models.Model):
+    source_name = models.CharField(max_length=50)
+    created_by = models.CharField(max_length=50, null=True)
+
+
+class Currencies(models.Model):
+    name = models.CharField(max_length=20)
+    currency_code = models.CharField(max_length=10)
+
+
+class Categories(models.Model):
+    category_name = models.CharField(max_length=50)
+    created_by = models.CharField(max_length=50, null=True)
+
+
+class User(models.Model):
+    name = models.CharField(max_length=50)
+    surname = models.CharField(max_length=50)
+    login = models.CharField(max_length=50)
+    email = models.EmailField(max_length=50)
+
+
+class Income(models.Model):
+    amount = models.IntegerField()
+    date = models.DateField()
+    source_id = models.ForeignKey(Sources, on_delete=models.DO_NOTHING)
+    user_id = models.ForeignKey(User, on_delete=models.DO_NOTHING)
+    currency_id = models.ForeignKey(Currencies, on_delete=models.DO_NOTHING)
+
+
+class Expenses(models.Model):
+    name = models.CharField(max_length=50)
+    amount = models.IntegerField()
+    date = models.DateField()
+    user_id = models.ForeignKey(User, on_delete=models.DO_NOTHING)
+    currency_id = models.ForeignKey(Currencies, on_delete=models.DO_NOTHING)
+    Category_id = models.ForeignKey(Categories, on_delete=models.DO_NOTHING)
```

Comment:
To nie jest Twój kod, co nie? Dobrze byłoby przynajmniej napisać, że jakiś kod w pull request wynika z tego, że te modele nie są jeszcze zmerge'owane